gpt4all-chat/src/database.cpp

```diff
@@ -1635,13 +1646,16 @@ bool Database::scanQueueInterrupted() const
 
 void Database::scanQueueBatch()
 {
-    m_scanDurationTimer.start();
-
     transaction();
 
+    m_scanDurationTimer.start();
+
     // scan for up to 100ms or until we run out of documents
```

Comment:
Don't put the exact ms as comment here as that will be confusing if we change the timeout and forget to change comment.